src/main/resources/com/google/api/codegen/clientconfig/client_gapic.yaml

```diff
@@ -0,0 +1,6 @@
+type: com.google.api.codegen.ConfigProto
+templates:
+- codegen_provider: com.google.api.codegen.clientconfig.ClientConfigGapicProvider
+  input_element_view: com.google.api.codegen.InterfaceView
+  snippet_files:
+  - main.snip
```

Comment:
I think it would be nice to set up this client config stuff so that other output formats could be included too, e.g. xml for C#. What classes/files would you have to add `Json` to later if we later added support for generating an xml client config file? I would consider adding `Json` to those things now.